Pipeline/SimplePackageDeploy/SimplePackageDeploy.groovy

```diff
@@ -0,0 +1,276 @@
+import groovy.transform.*
+import groovy.cli.commons.*
+import groovy.json.JsonSlurper
+
+/************************************************************************************
+ * NAME: SimplePackageDeploy.groovy
+ * 
+ * DESCRIPTION: Deploy the package created by PackageBuildOutputs.groovy to target 
+ * libraries
+ *  
+ * This is a simple deployment script to untar the deployment package created by 
+ * PackageBuildOutputs.groovy, read the BuildReport.json and copy the components
+ * to the respective environment target libraries.
+ * 
+ * As of now, this script does not perform any activation activities like a CICS 
+ * NEWCOPY or a DB2 BIND.  
+ * 
+ * INVOCATION: groovyz SimplePackageDeploy.groovy [parameters]
+ *
+ * PARAMETERS:
+ *
+ * --workDir <dir>             Absolute path to the package untar temporary folder
+ * --tarFileName <filename>    Name of the package tar file with path
+ * --hlq <hlq>                 HLQ of the target environment libraries for the deployment 
+ * --packageWithExtension      Flag to show the package contains extension (Optional) 
+ * 
+ * SimplePackageDeploy.properties:
+ * copyModeMap                 JSON string of deploy type and copy mode pairs
+ * targetLibLLQMap             JSON string of deploy type and target LLQ pairs 
+ *
+ * INTERNALS: 
+ *
+ *  1) Reads the options --workDir, --tarFileName, and --hlq from the command line. Pass the optional 
+ *     parm --packageWithExtension if the build package is created with addExtension option. 
+ *  2) Reads the options copyModeMap and targetLibLLQMap from the SimplePackageDeploy.properties 
+ *     file.
+ *  3) Creates a temporary package extract folder in the path provided using the 
+ *     --workDir option.
+ *  4) Untars the deployment package specified by --tarFileName to the temporary 
+ *     folder created in the previous step.
+ *  5) Reads the BuildReport.json file from the temporary folder.
+ *  6) Identifies all the components to be deployed from the BuildReport.json file.
+ *  7) Identifies deploy types for the components to be deployed from the 
+ *     BuildReport.json file.
+ *  8) Identifies the target libraries based on the deploy type by concatenating the
+ *     --hlq value with the correspoding LLQ value from the JSON string - targetLibLLQMap.
+ *  9) Identifies the copy mode based on the deploy type and corresponding copy mode 
+ *     from the JSON string - copyModeMap.
+ * 10) Copies the components using the copy mode to the respective target libraries.
+ * 11) Deletes the temporary package extract folder after deployment.
+ *
+ * RETURNS:
+ *    Exit Status 0       -  Success
+ *    Exit Status 1       -  Failure
+ *
+ ************************************************************************************/
+
+@Field Properties props = null
+props = parseInput(args)
+
+def startTime = new Date()
+props.startTime = startTime.format("yyyyMMdd.hhmmss.mmm")
+println("** SimplePackageDeploy start at $props.startTime")
+
```

Comment:
In the PackageBuildOutputs script we also print the parameters/configuration of the script.
```
** Properties at startup:
   verbose -> false
   copyModeMap -> ["COPYBOOK": "TEXT", "COPY": "TEXT", "DBRM": "BINARY", "LOAD": "LOAD"]
   packagingPropertiesFile -> /ZT01/var/dbb/extensions/dbb/Pipeline/PackageBuildOutputs/packageBuildOutputs.properties
   startTime -> 20221111.025209.052
   publish -> false
   workDir -> /u/dbehm/test-zapp-app-out/mortgageout/build.20221111.022101.021/
   buildReportOrder -> [/u/dbehm/test-zapp-app-out/mortgageout/build.20221111.022101.021//BuildReport.json]
   addExtension -> false
```
I would prefer if we have the same for this script as well.

Reply:
This is done.